pylint/checkers/strings.py

```diff
@@ -499,8 +497,13 @@ def _check_new_format_specifiers(self, node, fields, named):
                                          node=node)
                         break
 
+                inferred = previous.infer()
+                if inferred is astroid.Uninferable:
```

Comment:
This is so peculiar. I'm trying to reproduce it, but every time a debugger is involved, it stops reproducing.